packages/salesforcedx-vscode-lwc-next/.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://npm.lwcjs.org/
```

Comment:
I'm not sure if we need this since it should use the top-level .npmrc.

Reply:
Yup, removed